<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramDialog</class>
 <widget class="QDialog" name="HistogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Histogram data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Count data values in bins to calculate a histogram</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Datasets</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Input dataset expression</string>
        </property>
        <property name="buddy">
         <cstring>indataset</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="indataset">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Output bin height dataset name</string>
        </property>
        <property name="buddy">
         <cstring>outdataset</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="HistoryCombo" name="outdataset"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Output &amp;bin position dataset name</string>
        </property>
        <property name="buddy">
         <cstring>outbins</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="HistoryCombo" name="outbins"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Automatic bin parameters</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&amp;Number of bins</string>
          </property>
          <property name="buddy">
           <cstring>numbins</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="numbins">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&amp;Minimum value</string>
          </property>
          <property name="buddy">
           <cstring>minval</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="HistoryCombo" name="minval">
          <property name="toolTip">
           <string>Minimum value of lowest value bin or &quot;Auto&quot; to get from dataset</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Ma&amp;ximum value</string>
          </property>
          <property name="buddy">
           <cstring>maxval</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="HistoryCombo" name="maxval">
          <property name="toolTip">
           <string>Maximum value of highest value bin or &quot; Auto&quot; to get from dataset</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="HistoryCheck" name="logarithmic">
          <property name="text">
           <string>&amp;Logarithmic</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Manual bin boundaries</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="QPushButton" name="binadd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="binremove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="bingenerate">
          <property name="toolTip">
           <string>Generate bin boundaries from parameters</string>
          </property>
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QListView" name="binmanuals"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="HistoryGroupBox" name="methodGroup">
     <property name="title">
      <string>Calculate</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="counts">
        <property name="toolTip">
         <string>Count number of items in bin</string>
        </property>
        <property name="text">
         <string>&amp;Counts</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="density">
        <property name="toolTip">
         <string>Compute probability density in bins</string>
        </property>
        <property name="text">
         <string>&amp;Density</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fractions">
        <property name="toolTip">
         <string>Compute fraction of items in bin</string>
        </property>
        <property name="text">
         <string>&amp;Fractions</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="HistoryGroupBox" name="cumlGroup">
     <property name="title">
      <string>Cumulative</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="cumlOff">
        <property name="text">
         <string>Not cumulative</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cumlStoL">
        <property name="text">
         <string>Small to large</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cumlLtoS">
        <property name="text">
         <string>Large to small</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="HistoryCheck" name="errorBars">
     <property name="text">
      <string>Compute error bars (Gehrels Poisson approximation)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statuslabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryCombo</class>
   <extends>QComboBox</extends>
   <header>historycombo.h</header>
  </customwidget>
  <customwidget>
   <class>HistoryGroupBox</class>
   <extends>QGroupBox</extends>
   <header>historygroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HistoryCheck</class>
   <extends>QCheckBox</extends>
   <header>historycheck.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HistogramDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HistogramDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
